src/tools/npa-tool-cmdline.c

```diff
@@ -1695,6 +1702,11 @@ void update_multiple_arg(void *field, char ***orig_field,
     default:
       break;
     };
+    if (*((void **)field) == NULL) {
+      free(old_field);
+      fprintf(stderr, "Failed to allocate memory: aborting");
+      exit(1);
+    }
```

Comment:
This is a generated file. The fix needs to applied to [gengetopt](https://www.gnu.org/software/gengetopt/gengetopt.html), not here.

Reply:
Removed and rebased on current master. I will figure out how to properly fix it in gengetopt. Thanks.

Reply:
Landed here:
http://savannah.gnu.org/bugs/index.php?54761
Hope it is the right place.